Untitled-Engine/Src/Untitled/Core/Events/KeyCodes.h

```diff
@@ -0,0 +1,131 @@
+#define UE_KEY_4				52
+#define UE_KEY_5				53
+#define UE_KEY_6				54
+#define UE_KEY_7				55
+#define UE_KEY_8				56
+#define UE_KEY_9				57
+#define UE_KEY_SEMICOLON		59
+#define UE_KEY_EQUAL			61
+#define UE_KEY_A				65
+#define UE_KEY_B				66
+#define UE_KEY_C				67
+#define UE_KEY_D				68
+#define UE_KEY_E				69
+#define UE_KEY_F				70
+#define UE_KEY_G				71
+#define UE_KEY_H				72
+#define UE_KEY_I				73
+#define UE_KEY_J				74
+#define UE_KEY_K				75
+#define UE_KEY_L				76
+#define UE_KEY_M				77
+#define UE_KEY_N				78
+#define UE_KEY_O				79
+#define UE_KEY_P				80
+#define UE_KEY_Q				81
+#define UE_KEY_R				82
+#define UE_KEY_S				83
+#define UE_KEY_T				84
+#define UE_KEY_U				85
+#define UE_KEY_V				86
+#define UE_KEY_W				87
+#define UE_KEY_X				88
+#define UE_KEY_Y				89
+#define UE_KEY_Z				90
+#define UE_KEY_LEFT_BRACKET		91
+#define UE_KEY_BACKSLASH		92
+#define UE_KEY_RIGHT_BRACKET	93
+#define UE_KEY_GRAVE_ACCENT		96
+#define UE_KEY_WORLD_1			161
+#define UE_KEY_WORLD_2			162
+
+// Function keys
+#define UE_KEY_ESCAPE		256
+#define UE_KEY_ENTER		257
+#define UE_KEY_TAB			258
+#define UE_KEY_BACKSPACE	259
+#define UE_KEY_INSERT		260
+#define UE_KEY_DELETE		261
+#define UE_KEY_RIGHT		262
+#define UE_KEY_LEFT			263
+#define UE_KEY_DOWN			264
+#define UE_KEY_UP			265
+#define UE_KEY_PAGE_UP		266
+#define UE_KEY_PAGE_DOWN	267
+#define UE_KEY_HOME			268
+#define UE_KEY_END			269
+#define UE_KEY_CAPS_LOCK	280
+#define UE_KEY_SCROLL_LOCK	281
+#define UE_KEY_NUM_LOCK		282
+#define UE_KEY_PRINT_SCREEN	283
+#define UE_KEY_PAUSE		284
+#define UE_KEY_F1			290
+#define UE_KEY_F2			291
+#define UE_KEY_F3			292
+#define UE_KEY_F4			293
+#define UE_KEY_F5			294
+#define UE_KEY_F6			295
+#define UE_KEY_F7			296
+#define UE_KEY_F8			297
+#define UE_KEY_F9			298
+#define UE_KEY_F10			299
+#define UE_KEY_F11			300
+#define UE_KEY_F12			301
+#define UE_KEY_F13			302
+#define UE_KEY_F14			303
+#define UE_KEY_F15			304
+#define UE_KEY_F16			305
+#define UE_KEY_F17			306
+#define UE_KEY_F18			307
+#define UE_KEY_F19			308
+#define UE_KEY_F20			309
+#define UE_KEY_F21			310
+#define UE_KEY_F22			311
+#define UE_KEY_F23			312
+#define UE_KEY_F24			313
+#define UE_KEY_F25			314
+#define UE_KEY_KP_0			320
+#define UE_KEY_KP_1			321
+#define UE_KEY_KP_2			322
+#define UE_KEY_KP_3			323
+#define UE_KEY_KP_4			324
+#define UE_KEY_KP_5			325
+#define UE_KEY_KP_6			326
+#define UE_KEY_KP_7			327
+#define UE_KEY_KP_8			328
+#define UE_KEY_KP_9			329
+#define UE_KEY_KP_DECIMAL	330
+#define UE_KEY_KP_DIVIDE	331
+#define UE_KEY_KP_MULTIPLY	332
+#define UE_KEY_KP_SUBTRACT	333
+#define UE_KEY_KP_ADD		334
+#define UE_KEY_KP_ENTER		335
+#define UE_KEY_KP_EQUAL		336
+#define UE_KEY_LEFT_SHIFT	340
+#define UE_KEY_LEFT_CONTROL	341
+#define UE_KEY_LEFT_ALT		342
+#define UE_KEY_LEFT_SUPER	343 // Windows / CMD key
+#define UE_KEY_RIGHT_SHIFT	344
+#define UE_KEY_RIGHT_CONTRO	345
+#define UE_KEY_RIGHT_ALT	346
+#define UE_KEY_RIGHT_SUPER	347
+#define UE_KEY_MENU			348
```

Comment:
Can this mismatch of spacing be fixed?

Reply:
That is precisely why I space and not tab. Indents like these could be spaces keeping start of line as tabs